iaso/api/storage.py

```diff
@@ -309,7 +309,9 @@ def create(self, request):
 
                 performed_at = timestamp_to_utc_datetime(int(log_data["performed_at"]))
 
-                concerned_instances = Instance.objects.filter(uuid__in=log_data["instances"])
+                concerned_instances = Instance.objects.none()
```

Comment:
Not very important, but I think concerned is not the exact word (relevant could work) . Concerned is probably a translation of "concerné"

Reply:
You're absolutely right!